cmd/cluster/aws/create.go

```diff
@@ -163,6 +163,9 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 	exampleOptions.PublicZoneID = infra.PublicZoneID
 	exampleOptions.InfraID = infraID
 	exampleOptions.ExternalDNSDomain = opts.ExternalDNSDomain
+	if exampleOptions.EtcdStorageClass == "" {
```

Comment:
do we want to default the API
https://github.com/openshift/hypershift/blob/main/api/v1alpha1/hostedcluster_types.go#L197
https://github.com/openshift/hypershift/blob/main/api/v1alpha1/hostedcluster_types.go#L1451
>

Reply:
I don't think so since we only want to do this if the platform is AWS